### PR TITLE
GamesAppsCompatibility

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by isaak654 — CHANGES_REQUESTED** [review]
The table should be written in Markdown for uniformity reasons with [FeatureComparison.md](https://raw.githubusercontent.com/sandboxie-plus/sandboxie-docs/main/Content/FeatureComparison.md).

You can use various online generators, for example: https://tabletomarkdown.com/generate-markdown-table/

**Comment by ghost** [comment]
> The table should be written in Markdown for uniformity reasons with [FeatureComparison.md](https://raw.githubusercontent.com/sandboxie-plus/sandboxie-docs/main/Content/FeatureComparison.md).
> 
> You can use various online generators, for example: https://tabletomarkdown.com/generate-markdown-table/

done

File: Content/GamesAppsCompatibility.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+:pushpin: Table legend:<br>
```
Comment (isaak654) [inline Content/GamesAppsCompatibility.md:3]:
```suggestion
# Compatibility with games and applications

:pushpin: Table legend:<br>
```

This basically adds a meaningful title, I hope it goes well.

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
> The table should be written in Markdown for uniformity reasons with [FeatureComparison.md](https://raw.githubusercontent.com/sandboxie-plus/sandboxie-docs/main/Content/FeatureComparison.md).
> 
> You can use various online generators, for example: https://tabletomarkdown.com/generate-markdown-table/

Are you aware if there is any way of making issues/tickets being automatically highlighted in .md file? There are a few tickets mentioned in the file but none of them get highlighted the way they are in the original discussion thread.

**Comment by isaak654** [comment]
Unfortunately the issue references such as #3260 cannot be automatically highlighted in Markdown.

If you want to convert them quickly in the linked format, go to the [original discussion](https://github.com/sandboxie-plus/Sandboxie/discussions/2927), copy one of the links from there and when you're in the Edit page, paste the link in this way:

https://github.com/sandboxie-plus/sandboxie-docs/assets/12372772/70bda907-18ef-4a31-bbf8-33601dd7cc25

Then you can repeat this step for the remaining links taken from the original discussion.

**Comment by isaak654** [comment]
Unfortunately the issue references such as `#3260` cannot be automatically highlighted in Markdown.

If you want to convert them quickly in the linked format, go to the [original discussion](https://github.com/sandboxie-plus/Sandboxie/discussions/2927), copy one of the links from there and when you're in the Edit page, paste the link in this way:

https://github.com/sandboxie-plus/sandboxie-docs/assets/12372772/70bda907-18ef-4a31-bbf8-33601dd7cc25

Then you can repeat this step for the remaining links taken from the original discussion.

**Comment by ghost** [comment]
> Unfortunately the issue references such as `#3260` cannot be automatically highlighted in Markdown.
> 
> If you want to convert them quickly in the linked format, go to the [original discussion](https://github.com/sandboxie-plus/Sandboxie/discussions/2927), copy one of the links from there and when you're in the Edit page, paste the link in this way:
> 
>  firefox_bZ86cTmCd5.mp4 
> Then you can repeat this step for the remaining links taken from the original discussion.

Perhaps it would be better to simply put a link in the docs to the original discussion instead of just copy-pasting the table to a separate file?

**Comment by isaak654** [comment]
You've got a point there, but before putting a link to the discussion, could you add a short introduction?

**Comment by isaak654** [comment]
You've got a point, but before putting a link to the discussion, could you add a short introduction?

**Comment by ghost** [comment]
> You've got a point, but before putting a link to the discussion, could you add a short introduction?

I've just added a description to the original discussion. I think I can shut down this repo, right? I believe you can put the link to the original discussion in the right place in the docs without my involvement.

**Comment by isaak654** [comment]
In light of your considerations, my current idea is to create a specific paragraph in the [Known Conflicts](https://github.com/sandboxie-plus/sandboxie-docs/blob/main/Content/KnownConflicts.md) page. In my opinion, this has a lower priority than the other PRs, so in the meantime I'm going to convert this PR to draft as a reminder.

**Pull request closed without merging**